store/views.py

```diff
@@ -4,79 +4,122 @@
 from django.contrib.auth.decorators import login_required
 from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
-
-# Create your views here.
-
+from datetime import date
 def index(request):
     return render(request, 'store/index.html')
 
 def bookDetailView(request, bid):
     template_name = 'store/book_detail.html'
+    book = Book.objects.get(id=bid)
```

Comment:
This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.